packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -36,12 +36,12 @@ export class H5GroveApi extends DataProviderApi {
     super(filepath, { baseURL: url, ...axiosConfig });
   }
 
-  public async getEntity(path: string): Promise<ProvidedEntity> {
+  public override async getEntity(path: string): Promise<ProvidedEntity> {
```

Comment:
Abstract methods don't require this `override` keyword (since technically they are being _implemented_ not _overridden_). However, I do find it makes the class more readable as explained in this [TS issue](https://github.com/microsoft/TypeScript/issues/47250).